docs/releases/v3.5.0.md

```diff
@@ -2,7 +2,7 @@
 
 Welcome to the latest release of the `logging` module of [`Kubernetes Fury Distribution`](https://github.com/sighupio/fury-distribution) maintained by team SIGHUP.
 
-This update is a minor patch with a little fix on ingress fluentd outputs
+This update is a minor version that adds support for the Kubernetes version 1.30 and 1.31. It also introduces a **breaking change** with the new time series database for Loki [`TSDB`](https://grafana.com/docs/loki/v2.9.x/operations/storage/tsdb/) along with the new [`v13 schema`](https://grafana.com/docs/loki/v2.9.x/operations/storage/schema/). These changes will improve efficiency, speed and scalability of Loki clusters.
```

Comment:
IMO it is a breaking change since we are changing the time series database and schema but, for the user, if configured correctly during the update, it is a transparent change since Loki will start using the new `schema_config` automatically on the specified date and old data will always be available. So in the end it does not really break things and could be considered a minor. We can jump to a major version to stick to the rule `breaking change -> major release` if it means to be more consistent with the release versioning.

Reply:
Even though what you say is true, it is true only for a user of furyctl. For a user of the module (like the distro), this is a breaking change and should be noted like that.

We also bumped the major version in networking and ingress because we introduced breaking changes in the module, and also in both cases they don't affect end-users of furyctl but they do impact users of the modules themselves.

I'd say to bump also here to v4.0.0, just to make sure that is clear to everybody that if you don't pay attention data-loss is possible.

Reply:
You're right,seeing the changes under this perspective the bump to major is abundantly justified. I can do the bump to the v4 and rename the branches and PRs here and on the distro repo and also create the new tag for the rc, if you're ok with it.